src/Tooltip/index.js

```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import includes from 'lodash.includes'
 
-import glossary from '../../static/docs/glossary'
+import glossary from '../../public/static/docs/glossary'
```

Comment:
@iAdramelk @shcheklein Isn't it a little funny to import a static asset from src code? I know we talked about this change but I didn't realize this would be the result 😋

Reply:
In perfect world it, together with the `md` files, should be placed in the content folder in root directory and moved automatically to public folder at build time, but in `next.js` we would need to do it by hand and it's too bothersome.

Reply:
OK makes sense. I'll leave this review open for future ref.